src/api/api.service.ts

```diff
@@ -147,18 +147,26 @@ export class ApiService {
           );
         }
 
-        /* Checking if the new updated API name already exist. */
-        const apiNameExist = await this.apiRepo.findOne({
-          where: { name: updateApiDto.name },
-        });
+        console.log(updateApiDto.name);
```

Comment:
anytime you use console.logs for debugging and the likes try to clean it up before pushing it...